reference/api/search.md

```diff
@@ -476,28 +485,63 @@ You will get the following response with the **cropped text in the `_formatted`
 
 **Parameter**: `cropLength`
 **Expected value**: a positive integer
-**Default value**: `200`
+**Default value**: `10`
 
-Configures the number of characters to keep on each side of the matching query term when using the [`attributesToCrop`](#attributes-to-crop) parameter. Note that this means there can be up to `2 * cropLength` characters in the cropped field.
+Configures the number of words to keep around the matching query term when using the [`attributesToCrop`](#attributes-to-crop) parameter. This value includes the query terms contained in the cropped text, as well as terms configured as stop words.
```

Comment:
> This value includes the query terms contained in the cropped text, as well as terms configured as stop words.

What do you mean with this?

- stop words are counted as words
- Stop words can have their own cropping if I search for `the` for example and `the` is a stop word?
example : `...the...`

Reply:
Think it just means that stop words and query terms are counted as part of the `cropLength` value.

So if your query is `The Lord of the Rings` with a `cropLength` of 10 and you have an exact match in one of the fields, Meilisearch won't take 5 words on either side of the matched phrase, it will subtract the 5 matched terms from `cropLength` and take the remainder (5) on either side.

Reply:
I find the first sentence / definition to be misleading. "Configures the number of words to keep around the matching query term" seems to contradict the following sentence.

`Configures the total number of words to appear in the cropped value.` would be more accurate in my opinion

Reply:
@dichotommy is mostly correct in his interpretation, it's possible/likely that we wouldn't get the full sentence. 

I have revised the paragraph so it is clear that stop-words and query terms both count against the `cropLength` value. And to address @dichotommy's second point.